docs: improve documentation for prometheus metrics (#1861)
kyma-project/telemetry-manager: docs/user/04-metrics.md

For metrics ingestion to start automatically, use the annotations of the following table.
If an Istio sidecar is present, apply them to a Service that resolves your metrics port.

> By annotating the Service, all endpoints targeted by the service will be resolved and scraped by the Metric agent bypassing the service itself.

Reviewer: ```suggestion
By annotating the Service, all endpoints targeted by the Service are resolved and scraped by the Metric agent bypassing the Service itself.
```
